01week/javascripting/datatypes.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+/**
+Write a JavaScript program to display the current day and time.
+Write a JavaScript program to convert a number to a string.
+Write a JavaScript program to convert a string to the number.
+Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+Boolean
+Null
+Undefined
+Number
+NaN
+String
+Write a JavaScript program that adds 2 numbers together.
+Write a JavaScript program that runs only when 2 things are true.
+Write a JavaScript program that runs when 1 of 2 things are true.
+Write a JavaScript program that runs when both things are not true.
+**/
+
+function currentDate() {          // displays the current date and time
+  const date = new Date();
+  return date;
+}
+console.log(currentDate());
+
+function convertToString() {      // converts a number into a string
+  const number = 10;
+  const string = number.toString();
+  return string;
+}
+console.log(convertToString());
+
+function convertToNumber() {      // convert a string to a number
+  const string = '27';
+  const number = parseInt(string);
+  return number;
+}
+console.log(convertToNumber());
+
+function showMeTheData() {        // displays what data type each element is
+  console.log(typeof true);
```

Comment:
typeof is correct in this case however, you should rewrite the function to accept a parameter and return the typeof that parameter. This function as it is just console.logs a bunch of typeof statements, functions should always return something.